docs/conf.py

```diff
@@ -1,126 +1,48 @@
-from recommonmark.parser import CommonMarkParser
-
-# TO DO - Create and Implement Vyper Lexer
-# def setup(sphinx):
-#     sys.path.insert(0, os.path.abspath('./utils'))
-#     from SolidityLexer import SolidityLexer
-#     sphinx.add_lexer('Python', SolidityLexer())
-
 
-# -- General configuration ------------------------------------------------
-
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
-
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
 extensions = [
+    "sphinx_copybutton",
     "sphinx.ext.autodoc",
     "sphinx.ext.intersphinx",
 ]
 
-# Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
-
-# The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-source_suffix = [".rst", ".md"]
-
-# The master toctree document.
 master_doc = "toctree"
 
 # General information about the project.
 project = "Vyper"
 copyright = "2017-2024 CC-BY-4.0 Vyper Team"
 author = "Vyper Team (originally created by Vitalik Buterin)"
 
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-# The short X.Y version.
-version = ""
-# The full version, including alpha/beta/rc tags.
-release = ""
-
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = "python"
-
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
-
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = False
-
+language = "vyper"
 
 # -- Options for HTML output ----------------------------------------------
-
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = "sphinx_rtd_theme"
-
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
-#
-# html_theme_options = {}
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
+html_theme = "shibuya"
+html_theme_options = {
+    "accent_color": "plum",
```

Comment:
It is possible to use another color theme, see https://shibuya.lepture.com/customisation/colors/#accent-colors for available options.

I picked "plum" because it is the closer color to the new logo (and it actually renders pretty well, mostly with the dark theme).

I can test other colors, if you would like to.

Reply:
Hello! 

I just tried the colour palette from https://github.com/vyperlang/vyper-brand next to the radix palette you linked and I think Purple is the best match: 

<img width="837" alt="image" src="https://github.com/vyperlang/vyper/assets/80323528/5c6e5431-ab3d-4a5b-8d51-fbc97f01e949">

Reply:
The color was updated, and changes are live at https://vyper-tests.readthedocs.io. Thank you @pentcle!

Reply:
looks great, thank you @BoboTiG